cwf/cloud/go/services/cwf/obsidian/handlers/handlers_test.go

```diff
@@ -985,12 +985,8 @@ func seedCwfGateway(t *testing.T, id string, hwId string) {
 
 func seedCwfTier(t *testing.T, networkID string) {
 	// setup fixtures in backend
-	_, err := configurator.CreateEntities(
-		networkID,
-		[]configurator.NetworkEntity{
-			{Type: orc8r.UpgradeTierEntityType, Key: "t1"},
-		},
-		serdes.Entity,
-	)
+	_, err := configurator.CreateEntities(context2.Background(), networkID, []configurator.NetworkEntity{
```

Comment:
I think 
```
context2 "context"
"golang.org/x/net/context"
```
are the same. What about removing "golang.org/x/net/context" which is the old reference to the library? We already did something similar for feg

Reply:
I'm going to be replacing all usages of the deprecated `golang.org/x/net/context` package in a followup PR - this is just something that the refactoring engine in IDEA did.